gateway-tests/test.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+declare -A testCases=(
+    [complete_values]="--values test-values.yaml"
+)
+
+declare -A testCaseCharts=(
+    [complete_values]="../charts/gateway"
```

Comment:
Also, let's move all tests into a `test` directory first.

Reply:
I created `tests` and moved the gateway tests in - if that looks good to you i'll move the trino tests in as well.

Reusing the DB and any other common fixtures seems like a good idea. I considered it, but wanted to avoid conflicts while getting things working. Lets do it as a follow up